generator/src/main/java/com/scottlogic/deg/generator/generation/databags/DataBagValue.java

```diff
@@ -1,10 +1,13 @@
-package com.scottlogic.deg.generator;
+package com.scottlogic.deg.generator.generation.databags;
+
+import com.scottlogic.deg.generator.DataBagValueSource;
+import com.scottlogic.deg.generator.inputs.InvalidProfileException;
 
 import java.util.Objects;
 
 public class DataBagValue {
-    public final Object value;
-    public final String format;
+    final Object value;
```

Comment:
Package private to allow access by DataBag, without the need to break encapsulation with a value-accessor